bin/src/calc.dart

```diff
@@ -60,6 +60,9 @@ ArgParser _initializeParser() => new ArgParser(allowTrailingOptions: true)
   ..addFlag("help", help: "Prints this help", negatable: false)
   ..addOption("workers", help: "Number of workers for parsing", defaultsTo: "1")
   ..addOption("output", help: "Output file path")
+  ..addOption("packages",
+      help: 'Where to find the packages file, that is, "package:..." imports.',
+      defaultsTo: ".packages")
   ..addOption("package-root",
-      help: 'Where to find packages, that is, "package:..." imports.',
-      defaultsTo: "packages");
+      help: 'Ignored/Deprecated. Package directories are no longer supported.',
+      defaultsTo: ".packages");
```

Comment:
That might be correct. I'll also see to provide a better explanation text. I only left this in to have old implementations not break on the args level – they currently break script-wise, but at least silently, i.e. not bothering travis. Never change a (silently) failing system ;-)